src/GUI/UI/UIHouses.h

```diff
@@ -3,10 +3,17 @@
 #include <string>
 
 #include "GUI/GUIWindow.h"
-
-
 #include "UIHouseEnums.h"
 
+// Right hand side dialogue writing constants
+const int sideTextBoxWidth = 143;
+const int sideTextBoxPosX = 483;
+const int sideTextBoxPosZ = 334;
+const int sideTextBoxPosY = 113;
+const int sideTextBoxBodyTextHeight = 174;
+const int sideTextBoxBodyTextOffset = 138;
+const int sideTextBoxMaxSpacing = 32;
```

Comment:
Should we name them the same as we name other constants?

SIDE_TEXT_BOX_WIDTH, etc.

Reply:
Also, don't they need to be static?

Reply:
More like `constexpr`, yes.

Reply:
Good point.